boards/arm/nucleo_h723zg/board.cmake

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: Apache-2.0
+
+board_runner_args(openocd "--use-elf")
+board_runner_args(jlink "--device=STM32H723ZG" "--speed=4000")
+
+include(${ZEPHYR_BASE}/boards/common/openocd.board.cmake)
+include(${ZEPHYR_BASE}/boards/common/jlink.board.cmake)
```

Comment:
Please add the cubeprogrammer runner as this will be the another runner available for now.
Waiting for openocd support, I would even suggest making it default (first one) over jlink.

Reply:
Done